meadow-client/src/Semantics.purs

```diff
@@ -90,57 +107,10 @@ derive instance eqValue :: Eq Value
-  show (NegValue value1) = joinWith "" ["NegValue (", show value1, ")"]
-  show (AddValue value1 value2) = joinWith "" [ "AddValue ("
-                                              , show value1
-                                              , ") ("
-                                              , show value2
-                                              , ")"
-                                              ]
-  show (SubValue value1 value2) = joinWith "" [ "SubValue ("
-                                              , show value1
-                                              , ") ("
-                                              , show value2
-                                              , ")"
-                                              ]
-  show (MulValue value1 value2) = joinWith "" [ "MulValue ("
-                                              , show value1
-                                              , ") ("
-                                              , show value2
-                                              , ")"
-                                              ]
-  show (DivValue value1 value2 value3) = joinWith "" [ "DivValue ("
-                                                     , show value1
-                                                     , ") ("
-                                                     , show value2
-                                                     , ") ("
-                                                     , show value3
-                                                     , ")"
-                                                     ]
-  show (ModValue value1 value2 value3) = joinWith "" [ "ModValue ("
-                                                     , show value1
-                                                     , ") ("
-                                                     , show value2
-                                                     , ") ("
-                                                     , show value3
-                                                     , ")"
-                                                     ]
-  show (ValueFromChoice idChoice value) = joinWith "" [ "ValueFromChoice "
-                                                      , showIdChoice idChoice
-                                                      , " ("
-                                                      , show value
-                                                      , ")"
-                                                      ]
-  show (ValueFromOracle idOracle value) = joinWith "" [ "ValueFromOracle "
-                                                      , toString idOracle
-                                                      , " ("
-                                                      , show value
-                                                      , ")"
-                                                      ]
+  show v = genericShow v
```

Comment:
This will yield a different output. Are you sure we're happy with that?

Reply:
yes